client/services/FeedbackGeneratorService.js

```diff
@@ -20,13 +20,20 @@
 
 tie.factory('FeedbackGeneratorService', [
   'FeedbackObjectFactory', function(FeedbackObjectFactory) {
-    var jsToHumanReadable = function(jsVariable) {
+    var _jsToHumanReadable = function(jsVariable) {
       if (typeof jsVariable === 'string') {
         return '"' + jsVariable + '"';
       } else if (typeof jsVariable === 'number') {
         return String(jsVariable);
       } else if (typeof jsVariable === 'boolean') {
+        // TODO(sll): Change human-readable representation based on language.
         return jsVariable ? 'True' : 'False';
+      } else if (Array.isArray(jsVariable)) {
+        var variable_array = [];
```

Comment:
Naming is lowerCamelCase -- i.e. variableArray (or, more descriptively, something like humanReadableElements).

Also, if you like:

    var humanReadableElements = jsVariable.map(function(arrayElement) {
      return _jsToHumanReadable(arrayElement);
    });